packages/devextreme/js/ui/file_uploader.js

```diff
@@ -301,6 +301,7 @@ class FileUploader extends Editor {
         this._$fileInput.prop({
             multiple: this.option('multiple'),
             accept: this.option('accept'),
+            title: this.option('hint') || null,
```

Comment:
Why should we write `|| null` here?

Reply:
For consistence with https://github.com/DevExpress/DevExtreme/blob/23_2/packages/devextreme/js/ui/widget/ui.widget.js#L158

Reply:
I believe we should remove it or use nullish coalescing here. But ok, let's postpone it